client/src/test/java/nl/altindag/client/service/ApacheCXFWebClientServiceShould.java

```diff
@@ -0,0 +1,35 @@
+package nl.altindag.client.service;
+
+import static nl.altindag.client.ClientType.APACHE_CXF;
+import static nl.altindag.client.TestConstants.HTTP_URL;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import nl.altindag.client.ClientConfig;
+import nl.altindag.client.model.ClientResponse;
+import nl.altindag.client.util.MockServerTestHelper;
+import nl.altindag.client.util.SSLFactoryTestHelper;
+import nl.altindag.ssl.SSLFactory;
+
+@ExtendWith(MockitoExtension.class)
+class ApacheCXFWebClientServiceShould {
+
+    @Test
+    void createClientWithSslMaterial() throws Exception {
+        MockServerTestHelper.mockResponseForClient(APACHE_CXF);
```

Comment:
The MockServerTestHelper provides a real basic server during the unit test, so if you use this you don't need `@ExtendWith(MockitoExtension.class)` I would advise to either fully mock with Mockito and drop the MockServerTestHelper or use MockServerTestHelper and don't use Mockito at all. I think using the MockServerTestHelper is much easier, so I would recommend that one.